app/Http/routes.php

```diff
@@ -151,6 +151,7 @@
     Route::get('p/{posts}', ['as' => 'forum.posts.show', 'uses' => 'PostsController@show']);
     Route::get('posts/{posts}/raw', ['as' => 'forum.posts.raw', 'uses' => 'PostsController@raw']);
     Route::resource('posts', 'PostsController', ['only' => ['destroy', 'update', 'edit']]);
+    Route::post('posts/{posts}/hide', ['as' => 'forum.posts.hide', 'uses' => 'PostsController@hide']);
```

Comment:
mostly because the resource controller doesn't provide a sensible route for something like *hiding* a post (i mean i could theoretically add it to destroy as we want to shift away from actually deleting posts, but that just doesn't sound right)